.github/workflows/build-test.yml

```diff
@@ -0,0 +1,53 @@
+name: Check
+
+on:
+  pull_request:
+
+concurrency:
+  # Only run once for latest commit per ref and cancel other (previous) runs.
+  group: ci-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  check-code-style:
+    name: Code Style
+    uses: playframework/.github/.github/workflows/sbt.yml@v1
+    with:
+      cmd: sbt scalafmtCheckAll scalafmtSbtCheck
+
+#  MiMa is not enabled yet, since this library is marked as "API may change".
+#  check-binary-compatibility:
+#    name: Binary Compatibility
+#    uses: playframework/.github/.github/workflows/binary-check.yml@v1
+
+  check-docs:
+    name: Docs
+    uses: playframework/.github/.github/workflows/sbt.yml@v1
+    with:
+      cmd: sbt doc
+
+  check-headers:
+    name: Headers
+    uses: playframework/.github/.github/workflows/sbt.yml@v1
+    with:
+      cmd: sbt headerCheck test:headerCheck
+
+  tests:
+    name: Tests
+    needs: # Waiting more lightweight checks
+      - "check-code-style"
+      # - "check-binary-compatibility"
+      - "check-docs"
+    uses: playframework/.github/.github/workflows/sbt-matrix.yml@v1
+    with:
+      java: >-
+        [ "11", "8" ]
+      scala: >-
+        [ "2.12.15", "2.13.8" ]
+      cmd: sbt ++$SCALA_VERSION test
```

Comment:
Travis also ran `paradox`, should we do the same?

Reply:
Ah, good point!  I think this is published as part of the Akka docs, technically.  Makes sense.

I'm going to throw it in as the `check-docs` check, instead of `sbt doc`.  `sbt doc` wasn't even in the travis rule, from what I can tell.